ts/commands/SampleCommands.ts

```diff
@@ -0,0 +1,55 @@
+import * as caporal from 'caporal';
+
+import ConfigService from '../services/ConfigService';
+import SampleService from '../services/SampleService';
+
+const configService = new ConfigService();
+const sampleService = new SampleService();
+
+class SampleCommands {
+
+  public static addCommands() {
+
+    (caporal as any)
+      .command('metric results', 'Sample data when you have the Element and Metric IDs')
+      .option('--username', 'Metricly Username')
+      .option('--password', 'Metricly Password')
+      .option('--profile', 'Metricly profile', /.*/, 'default')
+      .option('--format', 'output format: text, json', ['table', 'csv', 'json'], 'table')
+      .argument('<elementId>', 'Element ID')
+      .argument('<metricId>', 'Metric ID')
+      .option('--duration', 'Duration (PT1H, PT1D, PT1W)')
+      .option('--startTime', 'ISO8601 start time')
```

Comment:
Any defaults for these? End date of now would be helpful.

Reply:
Parameters are not required, it returns the default from the API (which I'm not sure, is it 1hr of data?)